dwm1001/src/lib.rs

```diff
@@ -904,15 +908,15 @@ impl DW_IRQ {
             // Safe, as I don't believe this can interfere with the critical
             // section we're in.
             unsafe { nrf52::NVIC::unmask(Interrupt::GPIOTE); }
-            timer.enable_interrupt(nvic);
+            timer.enable_interrupt();
```

Comment:
You probably need to enable the appropriate timer interrupt in the nvic here, otherwise the timer wont fire (you need to enable in the peripheral, AND in the NVIC.

Reply:
aaah I assumed that'd happen automagically now. should've checked, will fix now, thx!